openhands/runtime/client/runtime.py

```diff
@@ -115,13 +122,16 @@ def __init__(
         env_vars: dict[str, str] | None = None,
     ):
         self.config = config
-        self._port = find_available_tcp_port()
-        self.api_url = f'http://{self.config.sandbox.api_hostname}:{self._port}'
+        self._host_port = 30000  # initial dummy value
+        self._container_port = 30001  # initial dummy value
+        self.api_url = (
+            f'http://{self.config.sandbox.api_hostname}:{self._container_port}'
+        )
         self.session = requests.Session()
-
         self.instance_id = (
             sid + '_' + str(uuid.uuid4()) if sid is not None else str(uuid.uuid4())
         )
```

Comment:
I wonder if this uuid is still necessary. sid includes an uuid the vast majority of the time, doesn't it? Speaking off-hand, from UI it does, from main.py it does, I *think* from evals it does (either uuid or pid), from the new cli maybe it doesn't. 🤔 

(just a thought, not a PR review - maybe we can look at it sometime, wdyt?)

Reply:
Remove? Don't want that. Rather shorten the uuid to e.g. first 12 characters. 😂